config/routes.rb

```diff
@@ -2,7 +2,9 @@
   namespace :api do
     namespace :v1 do
       get "/welcome", to: "welcome#index"
-      resources :items
+      resources :items do
```

Comment:
I don't mind using the `resources` keyword here and having nested routes. But I think it would be better if you restricted the routes to the ones that are currently being used. 

```
resources :items, only: [:index, :show, :whatever] do
  resources :reviews, only: [:index, :show, :whatever]
end
```